### DISCUSSION TITLE
OpenBB Terminal v2.2.0

### DISCUSSION THREAD

**Discussion by jmaslek** [OpenBB-finance/OpenBB#4012] [General]
## Thank you and welcome to our new contributors 🔥
@AlisonB319, @Bonganibendi, @Chavithra, @Drinkler, @JerBouma, @LarsKarbach, @MertNuhuz, @Rob-Rychs, @colin99d, @hjoaquim, @jmaslek, @jose-donato, @joshuabuildsthings, @luqmanbello, @martinb-bb, @montezdesousa, @northern-64bit, @noubre, @sergio-abu and @skhan59bbk

## What's new

Happy new year everyone!  Our first release of the year focuses on squashing bugs that were left over from 2022.  You may notice now that when loading in a ticker in `stocks`, there is no longer any additional information passed.  

## What's changed 🚀
* Fix M1 build (#4010) @hjoaquim
* Fix historical futures plot axes (#4006) @montezdesousa
* [FIX] Pytorch-lightning trainer conflict with TQDM progress bar (#4002) @martinb-bb
* Change .env loading priority (#4005) @montezdesousa
* Fix integration tests bugs (#3993) @montezdesousa
* Feature/release branch gitflow (#3998) @luqmanbello
* Add ticker to the load export file (#3997) @jmaslek
* Fix #3787 (#3966) @northern-64bit
* Hotfix/bug #3945 (#3963) @northern-64bit
* Fix the news command and making it more stable (#3965) @northern-64bit
* Add extra error handling for yf in etf (#3985) @jmaslek
* Bump yfinance to 0.2.4 and fix tests (#3977) @montezdesousa
* Update overlap_model.py - wrong typehint  (#3970) @sergio-abu
* Update vwap.md - wrong type hint  (#3971) @sergio-abu
* Update overlap_model.py (#3955) @sergio-abu
* Update sma.md (#3956) @sergio-abu
* Feature/macro convert (#3924) @MertNuhuz
* Minor Fix for branch name check (#3979) @luqmanbello
* Add multiple countries option to events (#3938) @montezdesousa
* Last Price Bug into Previous Close #3865 (#3949) @Bonganibendi
* Put yf.info behind an exchange flag on `load` (#3951) @jmaslek
* Fix portfolio returns (#3931) @montezdesousa
* Fixed failing badge not used anymore (#3932) @jose-donato
* Rewrite and skip yfinance tests (#3953) @montezdesousa
* GitFlow - improve check for changes (#3948) @luqmanbello
* Issue #3882 Fixed (#3885) @MertNuhuz
* broken link  (#3889) @Rob-Rychs
* Update installation guides with videos (#3903) @JerBouma
* Rebase develop? (#3935) @jmaslek
* Update dependency: grpcio (#3937) @Chavithra
* Hotfix/dots sources (#3928) @jose-donato
* Gitflow - Autodoc (#3922) @luqmanbello
* Integration tests refactor (#3781) @montezdesousa
* Add crypto dd functions to sdk (#3888) @jmaslek
* poetry add ccxt@latest (#3920) @jmaslek
* Feature/gitflow (#3921) @luqmanbello
* Feature/gitflow (#3914) @luqmanbello
* Update develop (#3913) @luqmanbello
* Feature/gitflow (#3911) @luqmanbello
* GItflow Implementations (#3909) @luqmanbello
* Main --> develop (#3906) @luqmanbello
* Gitflow Implementations (#3904) @luqmanbello
* Gitflow Implementations (#3902) @luqmanbello
* Fix plot look when using eval and add query to choices (#3881) (#3901) @luqmanbello
* Feature/gitflow (#3900) @luqmanbello
* Testing Gitflow (#3899) @luqmanbello
* Fix plot look when using eval and add query to choices (#3881) (#3898) @luqmanbello
* Fix Stocks/QA/beta (#3876) @montezdesousa
* Fix forex/candle with source AlphaVantage (#3877) @montezdesousa
* Adds historical end date on the ca menu (#3867) @jmaslek
* Fix emacross with Polygon as source (#3874) @montezdesousa
* Fixing timezone not updating within same terminal session (#3871) @AlisonB319
* Convert flake8 to ruff (#3855) @colin99d
* Added single block data from Blockchain.com (#3406) @noubre
* stocks/fa/analysis returns empty dataframe instead of error when no data (#3858) @skhan59bbk
* Dependancy updates: Darts  --> `0.23.0` & thepassiveinvestor --> `1.0.11` (#3866) @martinb-bb
* update_loaded made more readable (#3859) @MertNuhuz
* Adds command to check top hackernews stories (#3841) @jose-donato
* fixes #3843 eclect screen typo (#3844) @skhan59bbk
* Display daily returns as bar chart with color=red for negative return… (#3850) @LarsKarbach
* Change `openbb.stocks.fa.dcf` to `openbb.stocks.fa.dcfc` within Docs (#3862) @JerBouma
* Fix issue on loading stock (#3860) @hjoaquim
* Fixes ValueError in Cell 21. (#3804) @joshuabuildsthings
* Update blcp display name (#3808) @Drinkler
* Fix the greeks calculations of the chain command (#3812) @northern-64bit
* Fix links (#3817) @jose-donato
* New installer versions (#3814) @jose-donato

We are proud of our community contributors and staunch supporters of open-source ecosystems.
Help us promote our community by tagging `@openbb_finance` on Twitter with a link to your pull request,
and join our Discord server to chat about your contribution! We want to hear about your experience!

### Links 🦋
[Website](https://openbb.co/), [Twitter](https://twitter.com/openbb_finance), [Linkedin](https://www.linkedin.com/company/openbb-finance), [Instagram](https://www.instagram.com/openbb.finance/), [Reddit](https://www.reddit.com/r/openbb/), [Discord](https://discord.com/invite/xPHTuHCmuV)

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenBB-finance/OpenBBTerminal/releases/tag/v2.2.0'>OpenBB Terminal v2.2.0</a>.</em>

**Comment by Cobra0804** [comment]
Thank you for this. It seems it is still the V.2.1 version when I download

**Reply by jmaslek** [reply]
Yeah we may have accidentally pushed out the windows version still saying 2.1.0.

If you can run `stocks/load aapl` and it doesnt give an error -- you are on the right version :)
